src/graphql/fields.ts

```diff
@@ -5,17 +5,29 @@ export enum LinkType {
 	Web = "Link.web",
 }
 
+/**
+ * For link fields that haven't been filled
+ */
 export type EmptyLinkField = null;
 
+/**
+ * Links that point to Documents
+ */
 export interface FilledMinimalLinkToDocumentField {
 	_linkType: LinkType.Document | string;
 }
 
+/**
+ * Link that points to external website
+ */
 export interface FilledMinimalLinkToWebField {
 	_linkType: LinkType.Web | string;
 	url: string;
 }
 
+/**
+ * Link that points to  media
```

Comment:
Extra space before "media".

We could also explain what "media" is, like "Link that points to media (images, PDFs, or any file in the Media Library)"